documentation/gnmi/design/DASH.json

```diff
@@ -0,0 +1,724 @@
+{
+"qos-settings": [
```

Comment:
What is the expected behavior when the CPS or max flow limit is reached? For example,

- When CPS rate is reached, is it expected that new connections won’t be learned even though the overall CPS per DPU has not maxed out?
- When CPS or max flows are reached, should a per ENI counter be incremented to track the number of dropped flows due to max limit? Should all or a subset of the dropped flows be sent to the Control Plane/Controller?